### ISSUE TITLE
Train on custom datasets using Cellpose

### ISSUE THREAD

**Issue by fanxiaochen-123** [MouseLand/cellpose#419]
Hello, I used Cellpose for retraining in the custom data set, and the saved model after training is as follows. May I ask how I can use the saved model for reasoning? Or which model can I save to replace the original model?

2022-01-05 21:36:32,809 [INFO] Epoch 190, Time 43354.3s, Loss 0.6758, Loss Test 0.3793, LR 0.2000
2022-01-05 22:14:13,626 [INFO] Epoch 200, Time 45615.1s, Loss 0.6738, Loss Test 0.3603, LR 0.2000
2022-01-05 22:17:57,837 [INFO] saving network parameters to /mnt/hd1/samba/codes/ocr_codes/image_cyto/train/models/cellpose_residual_on_style_on_concatenation_off_train_2022_01_05_09_33_56.847476

The model suffix is.847476. How do I use this model?

**Comment by kevinjohncutler** [comment]
@fanxiaochen-123 you may specify the model using the full directory. In CLI, use `--pretrained_model` followed by your path. In a python script or Jupyter notebook, you may use your model path to initialize a model to evaluate on your images: 

For example:
``` js
modeldir = '/mnt/hd1/samba/codes/ocr_codes/image_cyto/train/models/cellpose_residual_on_style_on_concatenation_off_train_2022_01_05_09_33_56.847476'
model = models.CellposeModel(gpu=use_GPU, pretrained_model=modeldir, net_avg=False)
masks, flows, styles = model.eval(images,diameter=None)

```
Like you, I also typically do not train a size model and therefore set `diameter=None`. Note that built-in models use `model_type` instead of `pretrained_model`, should you ever need those.

**Comment by fanxiaochen-123** [comment]
@kevinjohncutler Thank you. I have one more question for you. The eight-neighborhood calculation in Cellpose refers to adding the values of the eight neighboring regions in addition to itself when calculating thermal diffusion. Is this to determine the boundary of thermal diffusion? Or are we going to add gradients in eight neighborhoods?

**Comment by fanxiaochen-123** [comment]
@kevinjohncutler As for the loss function, does Cellpose only use loss_FN in models.py for the loss function during training? Are L2(MSE) and Sigmoid losses not being used?

**Comment by carsen-stringer** [comment]
yes it uses the method `loss_fn` in `models.py`, this uses the `criterion` and `criterion2` in `core.py` by default, which are MSELoss and BCEWithLogitsLoss. 

for diffusion (to create flows for training in `dynamics.masks_to_flows`) it adds its own value along with its 8 neighbors
